src/app/core/entity-components/entity-list/list-filter/list-filter.component.html

```diff
@@ -1,30 +1,8 @@
-<div *ngIf="displayAsToggle">
-  <mat-button-toggle-group [name]="filterConfig.name" [value]="selectedOption">
-    <mat-button-toggle
-      *ngFor="let option of filterConfig.options"
-      [value]="option.key"
-      (click)="selectOption(option.key)"
-    >
-      {{ option.label }}
-    </mat-button-toggle>
-  </mat-button-toggle-group>
-</div>
-
-<div *ngIf="!displayAsToggle">
-  <mat-form-field>
-    <mat-label>{{ filterConfig.label || filterConfig.name }}</mat-label>
-    <mat-select
-      [id]="filterConfig.name"
-      [value]="selectedOption"
-      [placeholder]="filterConfig.name"
-    >
-      <mat-option
-        *ngFor="let option of filterConfig.options"
-        [value]="option.key"
-        (click)="selectOption(option.key)"
-      >
-        {{ option.label }}
-      </mat-option>
-    </mat-select>
-  </mat-form-field>
-</div>
+<app-filter
+  (selectedOptionChange)="selectOption($event)"
+  [selectedOption]="selectedOption"
+  [label]="filterConfig.label || filterConfig.name"
+  [data]="filterConfig.options"
```

Comment:
Here the data is the possible options of the filter but in the JSDoc of the component it says that this should be the data which should be filtered?